packages/inventory/src/EntityDetail.js

```diff
@@ -82,6 +93,38 @@ class EntityDetails extends Component {
                         }
                     </SplitItem>
                 }
+                { this.state.isModalOpen && (
+                    <DeleteModal
+                        handleModalToggle={this.handleModalToggle}
+                        isModalOpen={this.state.isModalOpen}
+                        currentSytem={entity}
+                        onConfirm={() => {
+                            let displayName;
+                            let removeSystems;
+                            if (Array.isArray(entity)) {
```

Comment:
No need to check if entity is array, this will always be single entity in detail, you can leave just the else path.